bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,514 @@
+- use channel rollerpos only if you need the inverted value, otherwise use the control channel and Item Type Number, see example
+- Short push and long push events require firmware version 1.5.6+.
+- The different devices have different types of power meters = different set of channels.
+
+Every device has a channel group "device" with the following channels:
+
+|Group     |Channel      |Type     |read-only|Desciption                                                                       |
+|----------|-------------|---------|---------|---------------------------------------------------------------------------------|
+|device    |uptime       |Number   |yes      |Number of seconds since the device was powered up                                |
+|          |lastUpdate   |DateTime |yes      |Timestamp of last update on any of the device's channel                          |
+|          |signal       |Number   |yes      |WiFi signal strength (RSSI)                                                      |
+|          |lastAlarm    |String   |yes      |Most recent alarm for health check                                               |
+
+
+
+### Event Channel
+
+The binding maps the inbound HTTP events from the device into trigger events for those enabled in the Thing Configuration (see above).
+
+|Event Type|Description|
+|------------|-----------------------------------------------------------------------------------------------------------------|
+|BTN_ON      |The Button was pressed                                                                                           |
+|BTN_OFF     |The Button was released                                                                                          |
+|SHORTPUSH   |A short push to the button was detected  (requires FW 1.5.6)                                                     |
+|SHORTPUSH   |A long push to the button was detected   (requires FW 1.5.6)                                                     |
+|OUT_ON      |Relais output is on                                                                                              |
+|OUT_OFF     |Relais output is off                                                                                             |
+|ROLLER_OPEN |Roller has reached the OPEN position and stopped.                                                                |
+|ROLLER_CLOSE|Roller has reached the CLOSED position and stopped.                                                              |
+|ROLLER_SRTOP|Roller was STOPped while moving (not in the OPEN or CLOSE position)                                              |
+|ALARM       |The health monitoring detected a alarm condition, check "Alarm Events" details.                                  |
+                                                                                                                               |
+
+The channel is implemented as a Trigger and could be used in a Rule:
```

Comment:
Have you anywhere mentioned the channel id of it?
```suggestion
The channel is implemented as an event channel and can be used in a Rule:
```

Reply:
sure: |          |event        |Trigger  |yes      |Triggers an event with a payload provinding more information (JSON               |

at the component level, e.g. for a relay

Wording doesn't make sense to me: The event mechanism is implemented as a Trigger channel (id event, see the component channel definitions, e.g. relay)

Reply:
I see I not to remove the json remark

Reply:
I see I not to remove the json remark

Reply:
I turn the alarm into a trigger channel with defined alarm types:
### Alarm Events

The binding provides health monitoring functions for the device.
When an alarm condition is detected the channel alarm gets triggered and provides one of the following alarm types:

|Event Type|Description|
|------------|----------------------------------------------------------------------|
|WEAK_SIGNAL |The RSSI signal is monitored every 10 minutes. An alarm is triggered when RSSI is < -80, which indicates an unstable connection. |
|RESTARTED   |The device has been restarted.  |
|OVER_TEMP   |The device is over heating, check installation and housing.|
|OVER_LOAD   |An over load condition has been detected, e.g. from the roller motor.|
|OVER_POWER  |Maximum allowed power was exceeded. The relay was turned off. |
|LOAD_ERROR  |Device reported a load problem. |

A new alarm will be triggered on a new condition or every 10 minutes if the condition persists.

Reply:
Ok, those event types for the different alarms make sense and are much better than any JSON.